python/ray/data/_internal/batcher.py

```diff
@@ -49,11 +49,20 @@ class Batcher(BatcherInterface):
     # zero-copy views, we sacrifice what should be a small performance hit for better
     # readability.
 
-    def __init__(self, batch_size: Optional[int]):
+    def __init__(self, batch_size: Optional[int], ensure_copy: bool = False):
+        """
+        Construct a batcher that yields batches of batch_sizes rows.
+
+        Args:
+            batch_size: The size of batches to yield.
+            ensure_copy: Whether the Batcher should guarantee that batches are copied
+                slice copies of the base blocks (not zero-copy views).
```

Comment:
nit: `copied slice copies` is kind of hard to read. Could we just say `batches are always copied from the underlying blocks (not zero-copy views)`

Reply:
Ah yep that's a typo, I'll fix that!